tests/unit/fem/test_quadinterpolator.cpp

```diff
@@ -142,50 +169,69 @@ static bool testQuadratureInterpolator(const int dim,
       Vector ne(vdim*ND*NE);
       REQUIRE(ne.Size() == VRN->Height());
       REQUIRE(VRN->Height() == VRL->Height());
+      // Full results
+      Vector vq_val_f(dim*NQ*NE), vq_der_f(vdim*dim*NQ*NE),
+             vq_det_f(NQ*NE),
+             vq_pdr_f(vdim*dim*NQ*NE);
+      // Tensor results
+      Vector vq_val_t(dim*NQ*NE), vq_der_t(vdim*dim*NQ*NE),
+             vq_det_t(NQ*NE),
+             vq_pdr_t(vdim*dim*NQ*NE);
       {
          // Full
          VRN->Mult(nodes, ne);
          vqi->DisableTensorProducts();
-         Vector vq_val_f(dim*NQ*NE), vq_der_f(vdim*dim*NQ*NE),
-                vq_det_f(NQ*NE),
-                vq_pdr_f(vdim*dim*NQ*NE);
 
          vqi->Values(ne, vq_val_f);
-         val[0] = vq_val_f*vq_val_f;
 
          vqi->Derivatives(ne, vq_der_f);
-         der[0] = vq_der_f*vq_der_f;
 
          vqi->Determinants(ne, vq_det_f);
-         det[0] = vq_det_f*vq_det_f;
 
          vqi->PhysDerivatives(ne, vq_pdr_f);
-         pdr[0] = vq_pdr_f*vq_pdr_f;
       }
       {
          // Tensor
          VRL->Mult(nodes, ne);
          vqi->EnableTensorProducts();
-         Vector vq_val_t(dim*NQ*NE), vq_der_t(vdim*dim*NQ*NE),
-                vq_det_t(NQ*NE),
-                vq_pdr_t(vdim*dim*NQ*NE);
 
          vqi->Values(ne, vq_val_t);
-         val[1] = vq_val_t*vq_val_t;
 
          vqi->Derivatives(ne, vq_der_t);
-         der[1] = vq_der_t*vq_der_t;
 
          vqi->Determinants(ne, vq_det_t);
-         det[1] = vq_det_t*vq_det_t;
 
          vqi->PhysDerivatives(ne, vq_pdr_t);
-         pdr[1] = vq_pdr_t*vq_pdr_t;
       }
-      REQUIRE(val[0] == MFEM_Approx(val[1]));
-      REQUIRE(der[0] == MFEM_Approx(der[1]));
-      REQUIRE(det[0] == MFEM_Approx(det[1]));
-      REQUIRE(pdr[0] == MFEM_Approx(pdr[1]));
+      double norm, rel_error;
+
+      norm = vq_val_f.Normlinf();
```

Comment:
Why switching for a CPU-only `max` that might discard most of the comparisons?

Reply:
I thought this is CPU-only test. Is it not?